curp/src/server/bg_tasks.rs

```diff
@@ -44,12 +50,11 @@ pub(super) async fn run_bg_tasks<C: Command + 'static, CE: 'static + CommandExec
 ) {
     // establish connection with other servers
     let others = state.read().others.clone();
-    let connects = rpc::try_connect(
-        others,
-        #[cfg(test)]
-        reachable,
-    )
-    .await;
+
+    #[cfg(test)]
+    let connects = Conn::try_connect_test(others, reachable).await;
```

Comment:
Are we going to replace "reachable" to the "injecture fun" in the later commits? If yes, please leave a comment here.

Reply:
Sure.